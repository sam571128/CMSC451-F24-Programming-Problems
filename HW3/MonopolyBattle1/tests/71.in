58 2337 385002833
41 16
1 58
31 40
9 47
23 43
6 35
32 38
48 51
10 43
44 17
25 48
57 32
39 23
43 18
13 44
35 51
28 32
19 26
54 36
32 53
28 53
21 38
18 41
29 8
13 2
14 58
35 19
10 35
30 34
15 48
53 9
27 51
8 18
3 33
6 30
47 5
37 18
28 22
20 13
38 44
57 14
33 37
58 49
2 49
54 28
27 48
38 29
42 18
56 46
10 22
12 52
36 4
53 15
18 40
22 4
17 51
37 17
30 2
35 54
31 13
51 18
31 9
7 28
55 58
27 49
29 45
17 47
58 3
58 31
28 31
18 10
39 48
31 43
9 38
47 37
40 31
16 53
41 9
52 7
14 39
46 45
10 5
10 47
3 38
39 52
31 15
56 39
39 15
34 36
11 27
55 23
44 46
49 12
53 39
12 8
25 39
10 54
4 36
44 27
6 13
10 12
47 45
45 32
45 34
13 41
46 3
54 51
23 46
38 36
44 5
6 22
35 32
36 15
39 2
34 51
43 33
13 1
16 5
38 13
34 55
4 38
35 7
10 31
1 50
48 47
7 50
37 56
31 52
27 47
9 51
34 4
10 34
14 37
25 13
45 46
28 3
27 13
47 54
27 57
22 7
58 29
1 30
41 26
16 3
50 24
17 18
54 4
1 48
15 58
34 43
57 31
52 47
7 55
17 27
20 58
18 54
39 9
40 20
43 39
14 12
7 20
7 1
48 52
57 19
12 27
25 20
24 36
25 11
35 50
6 56
33 56
50 17
47 1
33 42
1 47
41 18
48 7
47 52
30 8
53 12
13 10
21 30
54 22
40 30
52 40
48 11
18 48
47 8
6 29
17 48
25 10
20 8
49 49
35 34
40 18
44 19
30 6
10 46
8 55
8 54
24 10
24 8
46 54
10 9
36 39
46 19
21 10
38 26
12 14
1 54
39 47
56 33
40 27
18 42
39 29
12 31
2 4
35 21
4 14
41 42
15 9
46 14
42 25
25 35
34 35
44 36
12 37
48 25
43 46
33 14
27 58
19 50
30 58
21 31
5 6
21 56
52 23
6 47
17 12
9 29
10 14
34 18
7 19
11 48
28 6
36 23
34 17
15 31
58 46
45 42
45 58
46 43
7 17
35 27
14 21
52 27
10 41
28 18
37 58
30 1
25 4
44 29
11 40
32 55
52 45
47 32
49 15
11 37
46 42
50 32
43 54
48 50
31 32
31 38
34 9
26 17
52 50
28 49
22 37
5 16
13 16
45 10
8 13
15 34
55 3
15 50
14 26
3 15
55 46
42 8
33 21
11 16
56 27
33 19
43 14
34 46
7 57
11 49
26 41
47 18
28 48
29 16
5 48
51 15
13 45
36 14
29 15
23 38
10 37
23 56
15 53
40 50
27 32
2 13
31 30
18 53
54 29
16 36
15 16
14 19
27 1
31 58
9 22
36 22
12 49
34 44
54 18
26 23
44 55
8 15
52 57
46 4
13 13
25 27
11 31
29 46
34 40
18 22
32 21
54 42
34 29
1 7
9 19
47 36
40 23
4 37
52 17
10 4
30 5
41 51
43 23
52 15
17 37
46 41
54 2
11 30
28 19
7 52
34 19
8 16
35 49
47 26
20 49
33 2
9 7
10 55
10 11
29 7
8 38
20 25
50 34
15 47
29 34
47 48
41 7
33 47
14 35
41 32
39 12
14 56
45 17
5 32
49 53
51 16
40 29
4 47
36 57
5 47
49 32
55 18
25 40
31 31
42 21
39 37
35 52
11 56
15 25
10 30
20 29
5 1
31 36
12 44
51 47
4 52
37 10
55 44
5 39
51 13
34 48
26 58
16 55
41 44
43 58
24 6
17 24
50 9
17 10
27 50
20 17
12 5
39 21
29 4
16 1
12 40
43 57
16 17
48 10
47 30
58 5
58 10
17 30
45 51
11 57
39 45
7 12
41 20
56 22
35 33
1 38
56 18
18 39
27 41
31 12
56 45
44 30
20 22
26 42
21 36
20 51
9 50
11 4
17 20
1 52
53 37
14 34
50 36
52 14
48 49
30 47
36 47
11 42
39 26
10 10
36 31
34 50
31 47
17 19
37 35
19 9
31 50
55 7
54 43
3 52
57 6
24 55
55 26
35 10
40 42
2 43
19 15
50 18
23 25
56 11
13 46
28 28
33 45
41 19
18 47
10 39
27 7
6 12
44 51
5 12
7 40
14 25
32 32
47 33
50 58
37 44
52 39
46 51
26 4
13 38
5 53
45 24
20 14
11 26
7 8
56 51
3 17
49 13
49 16
7 33
5 7
34 24
49 21
10 53
33 23
21 58
19 30
18 23
1 2
4 29
2 17
43 9
38 45
35 18
36 35
33 44
32 10
10 2
15 38
47 40
34 53
18 9
11 34
2 53
51 6
18 6
1 37
8 53
21 39
2 44
25 5
19 12
3 54
34 23
47 57
50 7
29 43
36 26
18 56
5 36
7 25
26 21
16 23
50 14
55 37
7 6
32 37
55 22
28 11
40 57
11 39
15 37
43 32
57 7
32 31
28 46
23 35
52 11
22 33
3 5
51 32
11 13
30 23
49 6
54 21
14 18
43 13
12 58
32 49
9 23
39 49
38 38
5 46
31 45
42 46
19 6
30 29
37 55
36 12
43 55
30 17
57 40
11 51
50 1
19 14
51 28
6 17
22 38
49 2
47 47
3 49
28 57
32 26
24 42
54 37
22 41
52 54
53 10
28 21
39 43
18 5
16 12
13 17
49 22
31 44
12 35
47 56
42 15
49 54
49 23
13 51
32 34
1 49
18 32
31 5
51 23
27 43
18 8
11 21
6 24
55 8
27 45
54 17
48 28
37 28
16 25
44 47
36 30
32 12
8 51
16 51
10 28
10 17
25 37
16 42
27 16
1 13
27 17
54 57
37 48
24 29
2 22
28 34
28 5
48 5
4 54
34 15
58 45
47 55
24 18
36 1
39 4
56 4
48 29
42 58
54 19
23 12
49 11
48 40
14 17
46 20
46 32
43 45
27 3
12 56
41 22
12 45
23 24
37 5
14 42
29 27
47 27
20 31
13 40
7 35
33 22
3 55
26 35
24 50
30 10
10 38
18 4
11 38
33 28
7 53
41 45
15 4
5 29
22 26
32 19
6 1
42 20
26 39
4 23
16 41
57 9
35 35
44 9
23 57
9 42
24 20
51 1
30 39
1 5
39 17
58 9
43 12
20 20
8 20
17 21
51 53
7 46
22 10
44 54
29 3
44 10
15 14
10 1
57 22
32 5
38 58
47 12
6 34
13 25
57 21
8 14
43 25
15 45
25 2
5 33
3 40
50 29
6 33
9 11
51 33
39 11
30 19
42 16
48 15
2 48
7 34
14 36
47 13
56 36
3 30
48 46
15 5
38 56
27 12
46 16
10 20
6 3
44 16
30 32
53 14
51 45
7 22
36 58
43 11
45 30
35 25
19 55
36 10
43 6
21 40
3 12
51 24
36 46
17 23
45 1
16 52
38 12
41 8
19 29
20 2
4 5
47 42
1 22
34 58
7 4
15 6
35 45
20 44
13 12
8 28
24 33
29 2
1 25
47 14
2 12
34 37
18 16
6 27
37 39
31 37
33 27
5 10
50 25
48 26
45 50
46 29
14 44
22 49
38 47
46 47
53 55
47 15
50 19
54 56
51 57
56 37
54 16
28 39
42 42
18 50
40 33
28 36
11 19
11 15
20 53
14 40
23 36
23 44
10 18
46 10
17 22
47 16
21 17
42 52
13 36
16 56
58 21
58 13
57 33
13 48
51 26
1 57
30 7
24 12
51 46
7 43
39 32
32 30
20 27
24 57
53 5
18 37
5 9
37 54
44 8
16 40
58 25
49 39
28 51
39 25
27 11
48 58
38 54
32 13
12 50
23 4
45 5
15 10
58 2
56 29
56 9
40 9
2 21
17 34
56 19
46 15
53 27
42 17
16 43
35 37
54 31
44 15
33 51
36 9
2 36
25 41
27 53
13 15
26 30
29 37
8 2
16 45
53 25
53 18
38 52
23 52
7 23
8 52
14 51
50 51
1 32
11 5
25 55
11 35
35 30
36 55
37 24
1 44
45 33
4 25
21 4
28 7
7 47
23 45
20 26
41 23
13 18
31 8
15 13
43 34
29 55
4 33
53 17
51 27
32 1
34 32
48 55
45 8
53 22
28 43
20 38
29 5
17 46
18 12
33 24
51 4
25 23
2 29
24 34
40 54
40 6
5 18
36 21
43 5
30 57
2 26
10 29
5 38
12 3
38 16
10 32
47 28
36 3
13 22
29 56
56 30
57 27
43 51
37 34
53 30
53 50
41 46
57 20
38 48
32 35
3 39
51 21
24 31
6 39
40 52
30 35
2 20
47 19
53 28
22 39
28 2
1 1
57 34
9 54
40 32
28 55
50 48
52 29
49 42
54 45
22 5
5 42
27 40
29 47
48 57
9 10
8 58
53 23
6 51
5 57
34 21
26 16
3 34
32 25
48 16
8 36
20 6
23 48
10 57
21 27
36 44
53 54
16 34
47 39
20 23
24 7
58 7
1 23
11 50
10 15
35 56
6 4
33 40
23 49
34 42
46 34
52 37
48 13
10 3
16 8
3 32
48 39
19 35
36 18
8 34
53 49
25 56
33 35
33 25
56 17
30 40
8 26
5 54
12 38
41 58
22 30
52 25
44 58
27 8
27 15
23 27
53 2
50 39
55 5
26 57
29 24
5 15
46 49
13 23
38 1
25 14
49 20
34 7
37 50
47 29
20 1
53 31
42 40
50 20
41 14
35 53
4 24
15 28
38 14
56 58
5 17
13 32
45 57
34 47
27 27
54 24
37 26
19 1
31 51
53 4
26 46
25 53
3 50
15 29
23 14
33 39
11 36
15 40
20 24
12 26
42 28
2 40
31 10
6 44
58 14
38 20
13 53
27 56
24 14
26 10
22 56
54 52
1 11
45 52
26 11
18 57
19 47
1 39
12 23
4 6
47 4
9 28
2 16
33 57
43 17
52 35
4 40
46 56
7 51
23 53
4 51
45 18
9 52
5 44
38 18
48 45
29 41
24 15
3 43
52 30
22 53
4 22
24 58
41 25
41 4
27 52
8 21
17 43
46 52
11 24
19 36
52 31
24 2
8 22
55 55
44 41
18 18
29 29
24 56
49 30
8 39
30 4
55 57
26 15
19 41
16 14
45 48
53 42
26 18
55 11
54 6
23 7
32 6
19 21
29 42
24 1
21 22
41 49
50 12
55 47
19 2
12 6
34 2
11 18
10 40
5 28
13 47
42 39
41 17
13 31
29 49
21 35
3 9
50 5
50 40
46 9
36 40
50 26
5 26
22 14
52 42
25 19
26 44
41 43
19 54
1 33
36 13
39 14
49 8
49 24
14 8
50 44
25 43
18 45
45 40
35 41
50 30
24 3
49 52
46 37
38 55
2 47
5 4
44 26
5 40
16 49
29 17
29 6
17 28
49 36
51 5
16 9
32 23
45 55
45 22
52 2
20 40
20 47
21 49
26 3
16 13
24 22
28 50
29 10
51 38
34 12
5 58
19 32
56 40
16 19
45 47
58 18
15 36
55 31
14 27
21 28
42 4
11 55
6 26
27 14
30 37
51 8
40 55
40 38
1 19
57 39
43 44
55 49
16 18
48 3
38 28
17 9
14 1
19 13
35 57
30 46
18 19
10 50
6 46
42 43
2 56
2 31
42 54
32 56
1 6
2 6
39 58
25 17
23 30
10 58
21 44
8 9
36 43
46 1
37 57
6 8
46 5
42 41
12 51
30 54
55 54
29 58
22 50
10 48
43 21
9 13
8 19
58 19
34 3
46 30
22 27
53 20
20 19
44 23
8 27
29 28
12 41
23 55
40 26
52 51
25 50
45 2
42 49
53 53
55 33
8 56
45 54
55 28
6 45
56 10
28 54
9 44
43 38
41 37
4 11
23 5
51 42
51 58
4 20
38 32
53 47
40 17
53 16
47 38
39 34
18 21
2 27
9 32
17 50
44 22
45 28
51 9
17 54
32 46
1 51
26 9
40 44
50 35
43 28
24 26
24 25
4 27
9 30
58 55
17 36
49 35
3 37
57 53
56 56
54 44
17 35
5 37
24 54
14 5
43 47
38 4
35 6
52 10
34 49
37 52
19 4
54 46
4 10
37 42
22 54
15 43
25 16
54 53
49 31
11 46
35 28
46 40
26 32
21 8
20 5
3 36
53 56
3 58
19 5
50 4
23 13
9 24
21 37
51 22
1 42
31 23
41 47
44 48
28 52
21 19
52 48
50 2
21 3
35 8
45 15
19 3
31 11
4 56
49 41
8 32
40 24
20 33
44 14
56 47
4 57
11 17
57 16
56 24
27 37
23 47
13 57
48 54
53 11
52 12
17 13
13 7
6 41
50 49
21 12
18 24
4 55
27 18
14 7
16 31
58 38
40 10
57 10
54 27
3 45
30 13
14 43
12 48
39 33
17 33
3 31
37 21
5 5
8 46
33 3
37 47
58 26
29 21
32 15
20 15
32 50
28 41
34 14
21 2
39 10
8 8
4 41
11 32
11 53
40 35
42 50
19 40
8 24
56 6
15 44
40 2
55 53
39 19
7 24
39 51
2 52
28 9
29 9
25 54
7 39
8 23
27 54
12 30
55 34
47 20
53 38
26 45
29 39
16 7
25 18
24 19
49 55
33 13
48 8
21 20
28 29
20 52
24 4
16 29
35 2
23 26
24 48
26 22
48 33
12 7
5 51
27 34
56 23
12 2
36 34
17 17
14 57
54 35
40 12
51 35
52 1
29 40
15 49
35 38
47 51
6 49
52 5
18 3
53 8
45 56
52 33
27 26
11 1
28 15
9 6
10 36
56 52
10 7
56 20
20 36
56 1
42 6
6 28
6 21
12 55
4 15
9 34
46 21
58 58
53 13
58 4
35 23
35 9
38 7
38 50
21 43
43 2
21 23
13 35
28 45
20 32
1 55
2 7
58 54
35 1
58 48
50 47
44 13
53 43
3 3
15 26
42 33
54 23
22 18
52 18
22 8
36 45
34 8
57 58
51 11
22 13
57 44
52 19
16 11
50 42
7 27
36 41
10 42
39 28
48 32
43 31
42 36
1 26
50 8
3 4
28 35
38 11
42 11
7 37
54 54
20 54
13 6
19 57
11 22
37 40
4 16
43 56
47 53
37 33
43 3
53 51
23 23
6 52
37 12
49 43
15 20
2 15
28 58
40 39
48 1
28 44
39 39
16 24
42 9
45 27
3 46
5 35
33 7
1 8
54 5
5 41
30 42
56 50
44 21
23 9
56 35
49 27
32 58
42 56
56 57
55 2
53 29
6 37
28 37
33 5
21 55
7 56
2 35
45 31
17 45
1 53
52 58
51 52
34 5
7 13
42 34
40 3
6 48
27 4
50 38
55 43
39 31
50 21
39 44
25 33
9 48
19 46
58 39
48 53
54 32
41 13
18 14
46 33
37 38
44 25
14 49
19 28
25 28
11 10
13 28
40 1
31 53
33 12
33 33
20 45
5 8
5 50
44 24
49 18
5 31
6 57
54 25
35 58
40 49
12 42
6 42
13 52
30 49
2 37
26 27
32 41
53 36
19 31
11 3
58 15
23 32
34 16
22 6
11 47
28 20
50 55
3 21
5 23
56 48
47 9
9 35
29 19
34 38
32 33
22 28
52 20
4 58
42 48
42 2
27 23
38 10
48 20
44 50
37 23
32 27
52 21
42 44
18 1
20 18
21 7
23 50
25 47
29 11
30 9
29 33
8 40
13 26
20 4
16 27
12 20
34 31
42 27
22 9
2 23
33 50
28 47
50 31
35 4
16 47
19 38
1 45
13 56
2 18
1 10
8 11
4 19
42 14
18 28
8 50
41 52
21 33
50 43
56 34
28 30
49 44
1 14
39 22
40 43
55 25
24 37
43 22
48 19
6 9
54 58
4 35
5 22
8 48
55 20
35 13
54 40
46 23
2 5
32 45
10 19
18 2
57 52
7 2
39 18
1 17
45 53
45 38
12 34
39 1
39 24
30 22
43 1
33 52
55 40
47 11
39 36
10 26
41 21
49 4
47 25
34 57
4 46
45 21
48 4
39 20
7 58
4 4
7 31
32 20
41 34
45 14
5 55
24 21
25 44
10 13
49 50
55 27
18 26
47 17
20 21
20 7
10 24
37 30
15 41
18 36
23 41
58 44
30 28
39 42
53 40
48 37
31 4
2 30
25 52
35 24
47 34
31 42
46 13
20 55
25 9
2 58
7 54
53 46
32 22
51 19
15 56
54 49
42 35
38 27
48 48
37 31
41 40
25 49
56 8
3 19
36 36
2 55
18 35
36 27
31 21
5 3
5 14
57 41
25 3
44 2
40 45
55 10
30 43
8 4
24 46
31 55
27 36
7 26
37 11
44 43
41 41
38 9
49 9
39 5
56 54
14 41
8 3
28 38
56 15
6 16
1 36
33 34
40 48
44 42
17 39
15 54
31 24
25 36
57 23
8 12
14 23
24 44
3 23
33 55
30 18
58 50
24 49
24 16
28 16
51 14
6 11
38 49
20 43
38 42
52 36
21 13
47 35
46 28
25 8
17 57
44 33
48 6
39 53
15 7
2 28
35 47
1 3
33 58
12 54
2 2
42 12
15 27
14 52
46 48
42 7
35 11
12 11
3 25
58 40
33 26
27 10
6 10
46 53
26 36
16 46
5 21
28 8
43 20
40 53
8 47
9 16
40 4
27 55
9 57
13 42
7 15
57 8
38 33
34 25
26 29
50 16
40 36
22 52
50 28
57 28
13 30
48 21
9 25
40 28
26 51
2 9
17 38
8 45
49 10
5 49
27 5
11 41
43 48
48 9
26 43
46 12
26 5
15 8
2 51
9 14
8 30
55 16
41 10
49 26
34 28
7 42
9 20
25 42
42 31
52 3
37 16
52 8
54 55
13 9
52 49
54 3
8 1
21 34
54 20
5 2
46 26
32 43
17 14
21 5
29 30
3 47
18 51
15 18
22 48
30 14
5 52
23 11
1 40
50 45
43 43
40 14
56 2
41 39
31 16
22 44
50 37
16 44
19 8
25 26
45 9
2 25
36 37
39 56
44 1
52 6
53 6
26 28
56 12
6 50
9 33
16 4
55 51
2 10
3 27
11 2
41 30
39 57
26 14
7 14
5 24
16 39
40 16
9 27
55 17
17 26
32 3
27 33
8 37
38 6
25 21
23 10
39 30
24 17
49 56
23 29
51 20
2 57
48 12
35 40
6 7
37 49
50 46
24 45
50 53
11 12
10 56
31 1
20 57
51 51
15 42
11 29
49 58
20 9
49 14
44 52
38 15
27 2
1 35
22 40
46 24
23 42
18 52
33 30
44 6
31 33
33 38
57 3
9 18
50 3
31 54
44 40
32 4
28 26
1 18
21 29
26 40
23 22
50 33
17 4
49 33
45 11
11 28
14 54
25 30
21 26
46 25
36 56
14 31
15 23
51 30
2 3
57 48
55 13
28 14
12 33
3 28
25 32
21 45
41 6
18 27
6 5
25 38
33 16
35 16
14 13
23 18
12 24
29 18
33 17
51 12
55 56
10 33
23 20
28 1
41 11
3 6
9 55
9 9
3 57
10 51
32 2
53 1
13 14
43 29
11 8
12 9
19 33
15 24
15 39
19 44
26 26
57 1
22 1
24 47
15 15
45 16
54 10
6 43
7 44
52 26
19 11
18 15
35 26
24 11
6 25
25 25
24 9
57 13
30 55
57 47
24 53
26 12
39 13
42 22
30 51
21 51
24 52
22 25
15 52
20 16
2 54
28 25
26 56
56 41
32 39
56 49
3 8
31 6
46 36
34 27
31 49
51 29
24 5
27 9
7 9
27 35
17 40
37 37
50 57
24 32
35 12
46 58
35 42
18 38
58 16
42 55
8 7
58 32
53 35
43 16
44 28
13 49
2 8
12 47
19 20
33 4
34 26
40 58
25 22
4 3
8 25
19 25
16 58
6 32
32 47
30 20
56 16
58 8
44 56
4 48
44 7
43 36
38 22
19 27
14 6
39 27
55 32
45 41
25 15
16 37
36 17
32 54
29 26
40 25
28 24
12 46
44 31
3 2
37 46
21 15
17 32
30 12
57 11
19 52
40 13
27 21
9 37
32 17
44 53
9 43
55 38
27 42
58 36
16 54
17 25
54 33
33 18
11 6
51 48
22 21
31 7
6 58
58 20
37 9
10 45
6 31
27 24
32 36
42 10
45 44
22 20
11 33
26 34
8 17
42 37
54 14
31 22
4 45
12 13
26 25
12 22
15 30
4 7
56 43
26 53
13 29
56 14
6 54
41 12
25 51
21 16
47 22
6 53
34 39
50 52
52 46
9 4
26 19
7 7
30 26
14 20
13 54
45 7
3 1
48 24
54 13
4 50
13 8
46 7
37 1
37 2
