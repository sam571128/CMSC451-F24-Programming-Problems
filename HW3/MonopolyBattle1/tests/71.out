117 71 106 71 71 197 93 71 71 71 186 93 71 139 71 71 71 71 71 158 218 71 94 145 223 71 99 71 71 142 71 71 106 104 71 71 75 71 94 71 71 174 71 71 71 92 71 71 71 124 71 93 71 71 120 127 139 117 